index.html

```diff
@@ -99,7 +100,13 @@
     {
       "@context": "http://schema.org",
       "@type": "Organization",
-      "name": "PyDelhi"
+      "name": "PyDelhi",
+      "url": "http://pydelhi.org",
+      "description":"Python Delhi User Group",
+      "email": "pythondelhi(at)gmail.com",
```

Comment:
Let's not expose/publish this email, it's not supposed to be used any communication purposes.